.circleci/config.yml

```diff
@@ -137,44 +135,18 @@ commands:
           path: artifacts
 
 jobs:
-  checkout_code:
-    <<: *defaults
-    steps:
-      - *restore_repo
-      - checkout
-      - save_cache:
-          key: *repo_cache_key
-          paths:
-            - ~/amplify-swift
-
-  install_gems:
```

Comment:
do we need the gemfile and gemfile.lock anymore? if not, can they also be removed?

Reply:
We are still using `fastlane` in `release` step and canaries. These gem related files are still needed.